src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -2,22 +2,28 @@
 
 public class FIFORhymer extends DefaultCountingOutRhymer {
 
-	public DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+	private final DefaultCountingOutRhymer TEMP = new DefaultCountingOutRhymer();

```

Comment:
lowercase name - temp object is accessed and modified (final does not always mean a constant)